packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -480,22 +454,15 @@ export interface IGCRuntimeOptions {
  */
 export interface IGarbageCollectorConfigs {
 	/**
-	 * Tracks if GC is enabled for this document. GC may not be enabled for old documents created pre-GC.
+	 * Tracks if GC is allowed for this document. GC may not be allowed for old documents created pre-GC.
```

Comment:
```suggestion
	 * Tracks if GC is allowed for this document. GC may not be allowed for old documents created pre-GC.
	 * Note that if GC is allowed for a document, it will be enabled - GC cannot be enabled per-session.
```